glue/core/tests/test_hub.py

```diff
@@ -165,6 +165,44 @@ def test_invalid_init(self):
         assert exc.value.args[0] == ("Inputs must be HubListener, data, "
                                      "subset, or data collection objects")
 
+    @pytest.mark.parametrize('priorities', (False, True))
+    def test_handler_priorities(self, priorities):
```

Comment:
```suggestion
    def test_handler_priorities(self, priorities):
        """
        Test that handlers are called in order of descending priority, if set,
        in order subscribed to otherwise.
```
Just because it took me a little to figure that order out ;-)